presto-cli-rpm/pom.xml

```diff
@@ -0,0 +1,165 @@
+        <version>337-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>presto-cli-rpm</artifactId>
+    <name>presto-cli-rpm</name>
+    <packaging>rpm</packaging>
+
+    <properties>
+        <air.main.basedir>${project.parent.basedir}</air.main.basedir>
+
+        <cli.tar.package>presto-cli-provisio-${project.version}</cli.tar.package>
+    </properties>
+
+    <dependencies>
+        <!-- for testing -->
+        <dependency>
+            <groupId>org.testcontainers</groupId>
+            <artifactId>testcontainers</artifactId>
+            <scope>test</scope>
+        </dependency>
+
+        <dependency>
+            <groupId>org.testng</groupId>
+            <artifactId>testng</artifactId>
+            <scope>test</scope>
+        </dependency>
+    </dependencies>
+
+    <build>
+        <plugins>
+            <!-- the default RPM lifecycle does not include testCompile -->
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <goals>
+                            <goal>testCompile</goal>
+                        </goals>
+                    </execution>
+                </executions>
+            </plugin>
+
+            <plugin>
+                <groupId>ca.vanzyl.provisio.maven.plugins</groupId>
+                <artifactId>provisio-maven-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>unpack</id>
+                        <phase>prepare-package</phase>
+                        <goals>
+                            <goal>provision</goal>
+                        </goals>
+                        <configuration>
+                            <outputDirectory>${project.build.outputDirectory}</outputDirectory>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+
+            <!-- Build presto-cli rpm using the untarred artifacts -->
+            <plugin>
+                <groupId>io.airlift.maven.plugins</groupId>
+                <artifactId>redlinerpm-maven-plugin</artifactId>
+                <extensions>true</extensions>
+
+                <configuration>
+                    <performCheckingForExtraFiles>false</performCheckingForExtraFiles>
+
+                    <packages>
+                        <package>
+                            <name>presto-cli-rpm</name>
+                            <nameOverride>presto-cli-rpm-${project.version}.x86_64.rpm</nameOverride>
+                            <version>${project.version}</version>
+                            <release>1</release>
+
+                            <group>Applications/Databases</group>
+                            <description>Presto CLI RPM Package.</description>
+                            <architecture>x86_64</architecture>
```

Comment:
Presto support both x86_64 and aarch64. presto-cli should support both, too.
Fortunately, as far as I know, presto-cli.jar doesn't have dependencies to CPU architectures.
I assume we can select `noarch` inatead.

Reply:
Good point. I'll test on an aarch64 system with the changes and update accordingly.

Reply:
It works. Updated to noarch.